frontend/src/reviews/ReviewsList.tsx

```diff
@@ -6,38 +6,33 @@ import EmptyBlob from 'src/common/EmptyBlob'
 import Loading from 'src/common/Loading'
 import MessageAlert from 'src/MessageAlert'
 import ReviewItem from 'src/reviews/ReviewItem'
-import { Decision, ReviewRequestInterface } from 'types/interfaces'
+import { ReviewRequestInterface } from 'types/interfaces'
 
 type ReviewsListProps = {
-  kind?: 'release' | 'access' | 'all'
+  kind?: 'release' | 'access' | 'all' | 'archived'
```

Comment:
There's no need for 'all' anymore, should we remove it and make this a required prop?